.github/workflows/dockerhub.yml

```diff
@@ -11,7 +11,7 @@ name: Publish Docker image
 
 on:
   release:
-    types: [published]
+    types: [released]
```

Comment:
https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#release